StationFinderRadioNetwork.cpp

```diff
@@ -25,7 +29,7 @@ StationFinderRadioNetwork::StationFinderRadioNetwork()
     fIconLookupThread(0),
     fIconLookupList()
 {
-    serviceName.SetTo("Radio Network");
+    serviceName.SetTo(B_TRANSLATE("Radio Network"));
```

Comment:
This isn't a name, I think, or is it? If not, sentence casing.

```suggestion
    serviceName.SetTo(B_TRANSLATE("Radio network"));
```

Reply:
It might have been. On their website it states "Community Radio Browser". I'll use that, it makes more sense.